source/Security.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-constant-condition */
```

Comment:
Consider having a common interface to get input from the user and pass/move api related args/fns to appropriate modules